examples/tinywl-new/PrimaryOutput.qml

```diff
@@ -77,11 +77,16 @@ OutputItem {
         }
     }
 
-    Image {
+    // /home/uos/.cache/treeland/wallpaper/1000/x11-x/0/deepin-theme-1.jpg
+    Wallpaper {
         id: background
-        source: "file:///usr/share/wallpapers/deepin/desktop.jpg"
-        fillMode: Image.PreserveAspectCrop
+        userId: 1000
+        output: rootOutputItem.output
+
+        fillMode: Image.Tile
+        cache: false
         asynchronous: true
+        sourceSize: Qt.size(1920, 1080)
```

Comment:
Why is 1920 1080?

Reply:
This is the value of the requestsize of the image provider. If not set, it is (-1, -1)